api/models.py

```diff
@@ -1,16 +1,43 @@
 # api/models.py
 from django.db import models
-from django.contrib.auth.models import User
+#from django.contrib.auth.models import User
 from django.utils import timezone
 
 
+class User(models.Model):
+    username = models.CharField(max_length=255)
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+    email = models.CharField(max_length=255)
+    phone_number = models.CharField(max_length=15)
+    password = models.CharField(max_length=255)
+    role = models.CharField(max_length=15,
+                            choices=(("owner", "Business owner"),
+                                     ("employee", "Employee"),
+                                     ("topemployee", "Employee with elevated privileges"),
+                                     ("admin", "Admin")),
+                            default="employee")
+    team = models.ForeignKey("Team", related_name="members", on_delete=models.SET_NULL, null=True)
```

Comment:
User and Team should have m->m relation. Not 1->N. Single user can be member of multiple teams